driver/web/apis_bbs.go

```diff
@@ -207,3 +207,43 @@ func (h BBsAPIsHandler) SendMail(l *logs.Log, r *http.Request, claims *tokenauth
 
 	return l.HTTPResponseSuccess()
 }
+
+// AddRecipients add recipients to an existing message
+func (h BBsAPIsHandler) AddRecipients(l *logs.Log, r *http.Request, claims *tokenauth.Claims) logs.HTTPResponse {
+	params := mux.Vars(r)
+	messageID := params["message-id"]
+	if len(messageID) == 0 {
+		return l.HTTPResponseErrorData(logutils.StatusMissing, logutils.TypePathParam, logutils.StringArgs("id"), nil, http.StatusBadRequest, false)
+	}
+
+	read := getBoolQueryParam(r, "read")
+	mute := getBoolQueryParam(r, "mute")
+
+	recipient, err := h.app.BBs.BBsAddRecipients(l, messageID, claims.OrgID, claims.AppID, claims.Subject, mute, read)
```

Comment:
This is wrong as you set for account id the one from the claims. This API will be called from other BBs which means that the account id will be service account but not human account.

You have to use the body with these items:
```
[
{"mute":true/false, "account_id":""1234}
...
]
```